src/MinimumNodeService.cpp

```diff
@@ -96,6 +96,20 @@ void MinimumNodeService::checkModeChangeTimeout()
   }
 }
 
+void MinimumNodeService::heartbeat()
+{
+  if ((module_config->currentMode == MODE_NORMAL) && !noHeartbeat)
+  {
+    if ((millis() - lastHeartbeat) > heartRate)
```

Comment:
These two if-s can be combined. But actually I think it is easier to read this way.
I don't think combining them will yield different machine code.